catalog/systemd.catalog.in

```diff
@@ -144,7 +144,7 @@ might still be busy with completing start-up.
 
 Kernel start-up required @KERNEL_USEC@ microseconds.
 
-initrd start-up required @INITRD_USEC@ microseconds.
+Initrd start-up required @INITRD_USEC@ microseconds.
```

Comment:
uppercase "Initrd" is almost as weird as uppercase "Systemd"

Reply:
That's why names should start with a capital letter, so that sentences can be formed in the usual fashion. The iCraziness and Yahooness! is something that maybe works on ads, but is really really awkward in normal life ;) There is no perfect solution here, but the lowercase letter looks more bad than the uppercase.